scripts/drupal-backup-all

```diff
@@ -146,7 +155,7 @@ while [ $# -gt 0 ] ; do
       # Test to see if we have site aliases defined for the specified site.
       # TODO: We assume 'live' is remote and 'backup' is local.  Confirm this?
       site=$option
-      if [ -z "$(drush site-alias @$site.live 2>/dev/null)" ] || [ -z "$(drush site-alias @$site.backup 2>/dev/null)" ] ; then
+      if [ -z "$(drush site-alias @${site}.live 2>/dev/null)" ] || [ -z "$(drush site-alias @${site}.backup 2>/dev/null)" ] ; then
```

Comment:
Shell variables may contain spaces. So when referencing a variable:
```diff
- @$site.live
+ "@${site}.live"
```

Reply:
If you put a human in the equation (meaning "we know that value will never contain a space"), that person should be with the script 24/7!

Reply:
Yes, quoting variables is always good practice even when you know it's unnecessary. Drush does not allow alias names to contain spaces or special characters, though, so in this case we really do know that this variable doesn't need to be quoted.